vue/mixins/locales.js

```diff
@@ -0,0 +1,156 @@
+export const localeMixin = {
+    methods: {
+        /**
+         * Set the currently chosen locale.
+         *
+         * @param {String} locale  The ISO-15897 standard locale definition string to be set.
+         */
+        setLocale(locale) {
+            this.$store.commit("SET_LOCALE", locale);
+        },
+        /**
+         * Gets the currently chosen locale.
+         *
+         * @returns {String} The chosen locale. If it's not set, it will return the fallback
+         * locale instead.
+         */
+        getLocale() {
+            return (
+                this.$store.state.localePlugin.locale ||
+                this.$store.state.localePlugin.localeFallback
+            );
+        },
+        /**
+         * Sets the fallback locale.
+         *
+         * @param {String} locale  The ISO-15897 standard locale definition string to be set.
+         */
+        setLocaleFallback(locale) {
+            this.$store.commit("SET_LOCALE_FALLBACK", locale);
+        },
+        /**
+         * Gets the fallback locale.
+         *
+         * @returns {String} The fallback locale.
+         */
+        getLocaleFallback() {
+            return this.$store.state.localePlugin.localeFallback;
+        },
+        /**
+         * Set the locales information.
+         *
+         * @param {Object} locales  The locales information object. Example:
+         * { en_us: { "ripe_twitch.value_a.value_b": "Value Example" } }.
+         */
+        setLocales(locales = {}) {
+            this.$store.commit("SET_LOCALES", locales);
+        },
+        /**
+         * Gets the object with all the locales.
+         *
+         * @returns {Object} The object with all the locales.
+         */
+        getLocales() {
+            return this.$store.state.localePlugin.locales;
+        },
+        /**
+         * Appends new locales information to the specified locale. If the locale isn't
+         * specified, it will use the already chosen locale.
+         *
+         * @param {Object} locales  The locales information object. Example:
+         * { en_us: { "ripe_twitch.value_a.value_b": "Value Example" } }.
+         * @param {String} locale  The ISO-15897 standard locale definition string to be set.
+         */
+        addLocales(locales = {}, locale = null) {

```

Comment:
Make the locale mandatory. If you're adding something you should know what does it belong to

Reply:
As it is, if the locale isn't set, it will use the locale that is currently being used by the plugin, as in it will use `getLocale()`